extras/userdoc/md/documentation/part-2-populations-of-neurons.md

```diff
@@ -231,8 +224,7 @@ is often helpful to call the function `ResetNetwork()` within each loop
 iteration. It resets all nodes to their default configuration and wipes the data
 from recording devices.
 
-Command overview\[sec:Command-overview\]
---------------------------------------
+## Command overview\[sec:Command-overview\]
```

Comment:
Do we need the `[sec:Command-overview]` here? I thought, anchors are created automatically.